src/librustc_data_structures/sync.rs

```diff
@@ -314,7 +314,7 @@ cfg_if! {
         pub use parking_lot::MutexGuard as LockGuard;
         pub use parking_lot::MappedMutexGuard as MappedLockGuard;
 
-        pub use once_cell::sync::OnceCell;
+        pub use std::lazy::OnceCell;
```

Comment:
```suggestion
        pub use std::lazy::SyncOnceCell as OnceCell;
```

Reply:
Sorry about that, I was definitely not paying enough attention.

Reply:
Fixed!